File: Examples/VisualBasic/AsposePdfFacades/Working-Document/SetFileInfo.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports System
Imports Aspose.Pdf
Imports Aspose.Pdf.Facades
Namespace AsposePDFFacades.WorkingDocument
    Public Class SetFileInfo
        Public Shared Sub Run()
            ' ExStart:SetFileInfo
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfFacades_WorkingDocuments()
            ' Open document
            Dim fileInfo As New PdfFileInfo(dataDir & "SetFileInfo.pdf")
            ' Set PDF information
            fileInfo.Author = "Aspose"
            fileInfo.Title = "Hello World!"
            fileInfo.Keywords = "Peace and Development"
            fileInfo.Creator = "Aspose"
            ' Save updated file
            fileInfo.SaveNewInfo(dataDir & "SetFileInfo_out_.pdf")
            ' ExEnd:SetFileInfo

        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePdfFacades/Working-Document/SetFileInfo.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2014 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.Facades;
using System;

namespace Aspose.Pdf.Examples.CSharp.AsposePDFFacades.WorkingDocument
{
    public class SetFileInfo
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfFacades_WorkingDocuments();
            //open document
            PdfFileInfo fileInfo = new PdfFileInfo(dataDir+ "SetFileInfo.pdf");
            //set PDF information
            fileInfo.Author = "Aspose";
            fileInfo.Title = "Hello World!";
            fileInfo.Keywords = "Peace and Development";
            fileInfo.Creator = "Aspose";
            fileInfo.CreationDate = DateTime.Now.ToString();
            fileInfo.ModDate = DateTime.Now.ToString();
            //save updated file
            fileInfo.SaveNewInfo(dataDir+ "SetFileInfo_out.pdf");
            
            
        }
    }
}